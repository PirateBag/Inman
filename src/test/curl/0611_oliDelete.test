Assuming that W-001 was created in a prior test, delete it.

            {  "crudAction" : "CHANGE", "id" : 2, "itemId" : "2", "quantityOrdered" : "5", "quantityAssigned" : "0.0",
                "startDate" : "2025-0701", "completeDate" : "", "orderState" : "PLANNED",
                "debitCreditIndicator" : "ADDS_TO_BALANCE" }
===JSON===


{ "rows" : [
    {  "crudAction" : "DELETE", "id" : 1, "itemId" : "1", "quantityOrdered" : "1", "quantityAssigned" : "0.0",
        "startDate" : "2025-07-01", "completeDate" : "", "orderState" : "PLANNED",
        "orderType" : "MOHEAD" }
] }