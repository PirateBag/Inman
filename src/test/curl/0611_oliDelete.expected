{
  "responseType" : "MULTILINE",
  "errors" : [ ],
  "data" : [ {
    "id" : 1,
    "activityState" : "DELETE",
    "itemId" : 1,
    "quantityOrdered" : 1.0,
    "quantityAssigned" : 0.0,
    "startDate" : "2025-0701",
    "completeDate" : "2025-0703",
    "parentOliId" : 0,
    "orderState" : "PLANNED",
    "orderType" : "MO"
  } ]
}